docker/images/docker-spark/master/supervisord.conf

```diff
@@ -0,0 +1,39 @@
+[supervisord]
+logfile = /tmp/supervisord.log
+logfile_maxbytes = 50MB
+logfile_backups=10
+loglevel = info
+directory = /tmp
+nocleanup = true
+childlogdir = /tmp
+strip_ansi = false
+#environment = KEY1="value1",KEY2="value2"
+
+[program:ingestion]
+command=/usr/bin/java -cp /spark/conf:/spark/jars/*
+     -Xmx512m org.apache.spark.deploy.SparkSubmit
+     --conf spark.executor.memory=512m
+     --conf spark.driver.memory=512m
+     --conf spark.master=spark://spark-master:7077
+     --conf spark.driver.cores=1
+     --conf spark.cores.max=2
+     --conf spark.executor.cores=1
+     --queue default
+     --class com.scality.clueso.MetadataIngestionPipeline
+     --name 'Clueso Metadata Ingestion Pipeline'
+     file:///clueso/lib/clueso.jar /clueso/conf/application.conf
+directory=/
+#user=1000
+autostart=true
+autorestart=true
+process_name=metadata_ingestion
+numprocs=1
+stdout_logfile=/dev/fd/1
+stdout_logfile_maxbytes=0
```

Comment:
why?

Reply:
since the "log" is being redirected to stdout and 0 is unlimted in this context